python/cuml/test/test_fil.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+from cuml import FIL as fil
+from cuml.utils.import_utils import has_treelite, has_xgboost
+
+from sklearn.datasets import make_classification
+from sklearn.metrics import accuracy_score
+
+if has_treelite():
+    import treelite as tl
+
+if has_xgboost():
+    import xgboost as xgb
+
+
+def unit_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.unit)
+
+
+def quality_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.quality)
+
+
+def stress_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.stress)
+
+
+@pytest.mark.parametrize('n_rows', [unit_param(100), quality_param(1000),
```

Comment:
After this change, we should also add true unit tests. This is really an integration test and is fairly slow and has a lot of moving parts.

Some things to unit test would be:
 - Build a treelite tree with a known number of nodes, load it and confirm that we got it right
 - Vary the output types (sigmoid, average, raw) on a single model with a trivial example and confirm that we get the right outputs
 - Load a tree with unsupported features (learning to rank) and confirm that we get a nice clean expected exception
 - Load a forest with a degenerate shape, like single-tree, single-node and confirm we don't have issues.